crypto/signature.go

```diff
@@ -261,12 +261,22 @@ func verifySignature(
 				return time.Unix(verifyTime, 0)
 			}
 
+			seeker, ok := origText.(io.ReadSeeker)
+			if !ok {
+				return nil, newSignatureFailed(errors.New("message reader do not support seeking, cannot retry signature verification"))
+			}
+
+			_, err = seeker.Seek(0, io.SeekStart)
+			if err != nil {
+				return nil, newSignatureFailed(err)
```

Comment:
We should wrap errors with the context.
```suggestion
				err = errors.Wrap(err, "gopenpgp: could not rewind the data reader.")
				return nil, newSignatureFailed(err)
```
Can you also add this to the error from `signatureReader.Seek(0, io.SeekStart)` below ? It seems we forgot to wrap it.

Reply:
Thanks fixed!